app/actions/index.js

```diff
@@ -71,14 +73,17 @@ export const addDat = key => dispatch => {
     }
     updateProgress()
 
-    dat.network.on('connection', con => {
-      updateConnections()
-      updateState()
-      con.on('close', () => {
+    dat.events.on('join network', () => {
+      dat.network.on('connection', con => {
         updateConnections()
         updateState()
+        con.on('close', () => {
+          updateConnections()
+          updateState()
+        })
       })
     })
+    dat.events.emit('join network')
```

Comment:
this is for the case where `togglePause` joins the network again, we need to reattach the event listeners